assets/base.css

```diff
@@ -1191,37 +1191,71 @@ deferred-media {
   position: relative;
 }
 
+.share-button__button {
+  font-size: 1.4rem;
+  display: flex;
+  align-items: baseline;
+  color: rgba(var(--color-link),var(--alpha-link));
+  margin-left: 0;
+  padding-left: 0;
+
+}
+
+.share-button__button, .share-button__fallback button {
+  cursor: pointer;
+  background: var(--color-background);
+  border: none;
+}
+
 .share-button__button .icon-share {
```

Comment:
When I am tabbing and hovering, there seems to be some overlapping of the rings: 

<img width="384" alt="Screen Shot 2021-08-09 at 2 45 16 PM" src="https://user-images.githubusercontent.com/19962891/128757829-02373474-6916-4523-af66-bad981274947.png">

Reply:
But I don't think that should be an issue no ? If you use a mouse you probably don't tab and hover different things when using a site 🤔

Reply:
Yeah from what I was testing it works this way when you're just tabbing through. The only way I was able to replicate what's in the screenshot above was by navigating with the `tab` key to the copy icon and then hovering with my mouse the input: [video](https://screenshot.click/21-08-que69-75l2j.mp4)

Reply:
Ah I see, that's fine to leave 👍